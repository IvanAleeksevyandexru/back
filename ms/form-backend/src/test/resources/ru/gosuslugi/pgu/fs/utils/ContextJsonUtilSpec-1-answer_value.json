{
  "regAddr": {
    "region": "Приморский",
    "city": "Владивосток",
    "district": "",
    "town": "",
    "inCityDist": "",
    "street": "Прапорщика Комарова",
    "additionalArea": "",
    "additionalStreet": "",
    "house": "48",
    "houseCheckbox": false,
    "houseCheckboxClosed": false,
    "building1": "",
    "building2": "",
    "apartment": "25",
    "apartmentCheckbox": false,
    "apartmentCheckboxClosed": false,
    "index": "690091",
    "geoLat": "43.1205736",
    "geoLon": "131.892925",
    "fullAddress": "690091, край. Приморский, г. Владивосток, ул. Прапорщика Комарова, д. 48, кв. 25",
    "addressStr": "край. Приморский, г. Владивосток, ул. Прапорщика Комарова",
    "lat": "43.1205736",
    "lng": "131.892925",
    "okato": "05401364000",
    "hasErrors": 0,
    "fiasCode": "21294388-e1c5-417e-bd32-622705a79d2f",
    "apartmentFias":"",
    "regionFias": "",
    "streetFias": "",
    "houseFias": "",
    "townFias": "",
    "kladrCode": "77000000000252200",
    "apartmentKladr":"77000000000252200",
    "streetKladr": "77000000000252200",
    "regionKladr": "7700000000000",
    "townKladr": null,
    "houseKladr": null,
    "postalCode": null,
    "regionCode": "25"
  },
  "regDate": "28.10.2020"
}