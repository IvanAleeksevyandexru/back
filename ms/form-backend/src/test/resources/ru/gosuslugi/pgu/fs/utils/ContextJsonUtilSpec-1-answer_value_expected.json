{
  "regAddr": {
    "additionalArea": "",
    "additionalAreaShortType": "",
    "additionalAreaType": "",
    "additionalStreet": "",
    "additionalStreetShortType": "",
    "additionalStreetType": "",
    "addressStr": "край. Приморский, г. Владивосток, ул. Прапорщика Комарова",
    "apartment": "25",
    "apartmentCheckbox": false,
    "apartmentCheckboxClosed": false,
    "apartmentShortType": "кв",
    "apartmentType": "квартира",
    "building1": "",
    "building1ShortType": "",
    "building1Type": "",
    "building2": "",
    "building2ShortType": "",
    "building2Type": "",
    "city": "Владивосток",
    "cityShortType": "г",
    "cityType": "город",
    "district": "",
    "districtShortType": "",
    "districtType": "",
    "fullAddress": "690091, край. Приморский, г. Владивосток, ул. Прапорщика Комарова, д. 48, кв. 25",
    "geoLat": "43.1205736",
    "geoLon": "131.892925",
    "hasErrors": 0,
    "house": "48",
    "houseCheckbox": false,
    "houseCheckboxClosed": false,
    "houseShortType": "д",
    "houseType": "дом",
    "inCityDist": "",
    "inCityDistShortType": "",
    "inCityDistType": "",
    "index": "690091",
    "fiasCode": "21294388-e1c5-417e-bd32-622705a79d2f",
    "apartmentFias":"",
    "regionFias": "",
    "streetFias": "",
    "houseFias": "",
    "townFias": "",
    "kladrCode": "77000000000252200",
    "apartmentKladr":"77000000000252200",
    "streetKladr": "77000000000252200",
    "regionKladr": "7700000000000",
    "townKladr": "",
    "houseKladr": "",
    "postalCode": "",
    "oktmoName": "",
    "oktmo": "",
    "lat": "43.1205736",
    "lng": "131.892925",
    "okato": "05401364000",
    "region": "Приморский",
    "regionCode": "25",
    "regionShortType": "обл",
    "regionType": "область",
    "street": "Прапорщика Комарова",
    "streetShortType": "ул",
    "streetType": "улица",
    "town": "",
    "townShortType": "",
    "townType": ""
  },
  "regDate": "28.10.2020"
}